convert_matrices.py

```diff
@@ -968,13 +970,18 @@ class GSE127969(Converter):
     """
 
     def _convert(self):
-        # Fails with a UnicodeError due to fancy quotation marks (0x93 0x94)
-        raise PostponedImplementationError('https://github.com/DailyDreaming/load-project/issues/114')
-        # noinspection PyUnreachableCode
         self._convert_matrices(
-            CSV('GSE127969_counts_TPM_ALL.csv.gz', sep='\t')
+            CSV(
+                'GSE127969_counts_TPM_ALL.csv.gz',
+                sep='\t',
+                row_filter=self._filter,
+                kwargs=dict(encoding='latin-1'))
         )
 
+    def _filter(self, row: List[str]):
+        if row[0].startswith('\x93') or row[0].endswith('\x94'):
```

Comment:
Are you sure that the only occurrences of these characters is at the beginning or end of a cell?

Are you sure that they only occur in the first column?

If the answer to either to both of these is yes, start thinking about how you can optimize statements: For example, if you know a character occurs at the beginning of a string, do you really need to scan the entire string? Same with the end. If the argument to startswith() is a single character string, can you optimize that expression?

If then answer to either of these question is no, we need to rethink this approach.